R/print.R

```diff
@@ -42,7 +42,7 @@ print.gsplot <- function(x, ...){
 
   top.par <- par(no.readonly=T)
   for (i in which(names(views) %in% 'view')){
-    par(top.par)
+    # par(top.par)
```

Comment:
I don't understand why this was the issue w/ layout. This is a loop within 1 gsplot object. layout was calling 3 or 4 different gsplot objects.